frontend/test/__support__/cypress.js

```diff
@@ -304,3 +304,15 @@ function addQADatabase(engine, db_display_name, port) {
     expect(status).to.equal(200);
   });
 }
+
+export function adhocQuestionHash(question) {
+  if (question.display) {
+    // without "locking" the display, the QB will run its picking logic and override the setting
+    question = Object.assign({}, question, { displayIsLocked: true });
+  }
+  return btoa(unescape(encodeURIComponent(JSON.stringify(question))));
+}
+
+export function visitQuestionAdhoc(question) {
+  cy.visit("/question#" + adhocQuestionHash(question));
+}
```

Comment:
This was already present in `cypress.js` before, right? Did you cherry-picked it or copied it over from some other branch?

Reply:
The `visitQuestionAdhoc` function is already in `cypress.js` on master but not in the base branch here. This splits out the `adhocQuestionHash` function. That change will probably cause a (quickly resolved) merge conflict when this makes it back into master.